site/src/pages/TasksPage/TaskPrompt.tsx

```diff
@@ -225,7 +218,7 @@ const CreateTaskForm: FC<CreateTaskFormProps> = ({ templates, onSuccess }) => {
 			{externalAuthError && <ErrorAlert error={externalAuthError} />}
 
 			<fieldset
-				className="border border-border border-solid rounded-lg p-4"
+				className="border border-border border-solid rounded-3xl p-3 bg-surface-secondary"
```

Comment:
Nit: Not the biggest fan of the extra rounding. Feels inconsistent with the other elements on the page. I'm not a designer though.

Reply:
I see what you're saying. I *personally* prefer this one but again: not a designer haha.

Reply:
You may not be a designer, but I know you’ve got good taste. I’ll create a poll and see what others think haha.